neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -409,3 +419,60 @@ def list_address(self):
                 "watchonly": addr.IsWatchOnly,
             })
         return result
+
+    def send_many(self, params):
+        if len(params) not in [1, 2, 3] or type(params[0]) is not list:
+            raise JsonRpcError(-32602, "Invalid params")
+        output = []
+        for info in params[0]:
+            asset = get_asset_id(self.wallet, info['asset'])
+            if not type(asset) is UInt256:
+                raise JsonRpcError(-32602, "Invalid params")
+            address = info["address"]
+            try:
+                address = self.wallet.ToScriptHash(address)
+            except Exception:
+                raise JsonRpcError(-32602, "Invalid params")
+            amount = Fixed8.TryParse(info["value"], require_positive=True)
+            if not amount:
+                raise JsonRpcError(-32602, "Invalid params")
+            tx_output = TransactionOutput(AssetId=asset,
+                                          Value=amount,
+                                          script_hash=address)
+            output.append(tx_output)
+
+        standard_contract = self.wallet.GetStandardAddress()
+        signer_contract = self.wallet.GetContract(standard_contract)
+
+        contract_tx = ContractTransaction(outputs=output)
+
+        fee = Fixed8.TryParse(params[1]) if len(params) in [2, 3] else Fixed8.Zero()
+        if fee < Fixed8.Zero():
+            raise JsonRpcError(-32602, "Invalid params")
+
+        change_addr = params[2] if len(params) == 3 else self.wallet.Addresses[0]
+        try:
+            change_addr = self.wallet.ToScriptHash(change_addr)
+        except Exception:
+                raise JsonRpcError(-32602, "Invalid params")
+
+        tx = self.wallet.MakeTransaction(tx=contract_tx,
+                                         change_address=change_addr,
+                                         fee=fee)
+        if tx is None:
+            raise JsonRpcError(-300, "Insufficient funds")
+        data = standard_contract.Data
+        tx.Attributes = [
+            TransactionAttribute(usage=TransactionAttributeUsage.Script,
+                                 data=data)
+        ]
+        context = ContractParametersContext(
+            tx, isMultiSig=signer_contract.IsMultiSigContract
+        )
+        self.wallet.Sign(context)
+        if context.Completed:
```

Comment:
please add a test for both branches

Reply:
@ixje Done. Please see code as it appears here:
https://github.com/jseagrave21/neo-python/blob/47fa65d6bc3d446da8a63fa2cfb57ab1ee599d77/neo/api/JSONRPC/test_json_rpc_api.py#L1442